web/package.json

```diff
@@ -1,20 +1,45 @@
 {
-  "name": "platform.web",
-  "private": true,
-  "version": "1.0.0",
-  "description": "",
+  "name": "colark-platform",
+  "version": "0.0.1",
+  "description": "The monorepo for the Colark platform",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "keywords": [],
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/colark/platform.git"
+  },
+  "author": "Colark Inc.",
   "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/colark/platform/issues"
+  },
+  "homepage": "https://github.com/colark/platform#readme",
   "dependencies": {
-    "react-helmet": "^5.2.0",
-    "react-mailto": "^0.4.0",
-    "react-scrollchor": "^6.0.0",
```

Comment:
We still need scrollchor and the slick imports here (but mailto can be removed).  Please change this back; I think you meant to put these changes in platform/package.json; this is platform/web/package.json.